lightkube/core/internal_models.py

```diff
@@ -1,7 +1,7 @@
 import sys
 
 try:
-    from ..models import meta_v1, autoscaling_v1
+    from ..models import meta_v1, autoscaling_v1, apiextensions_v1
```

Comment:
In case of import failure we need to define a mock for each method/class used in apiextensions_v1. See bottom of this same file for examples.